backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You restore application backups by creating a xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-cr_restoring-applications[`Restore` custom resources (CRs)].
+You restore application backups by creating a xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-cr_restoring-applications[`Restore` custom resources (CRs)] using OADP.
```

Comment:
Same suggestion as my initial comment. I think all the other links should follow the same pattern as the link text does not natch the target heading and the formatting of the text does not indicate a hyperlink.